components/brave_wallet_ui/components/extension/connections/components/dapp_connection_networks/dapp_connection_networks.tsx

```diff
@@ -30,24 +31,29 @@ import {
 import { Text, Row, Column, ScrollableColumn } from '../../../../shared/style'
 
 interface Props {
-  coin: BraveWallet.CoinType
+  accountId: BraveWallet.AccountId | undefined
   selectedNetwork?: BraveWallet.NetworkInfo
   onChangeNetwork: (network: BraveWallet.NetworkInfo) => void
 }
 
 export const DAppConnectionNetworks = (props: Props) => {
-  const { coin, selectedNetwork, onChangeNetwork } = props
+  const { accountId, selectedNetwork, onChangeNetwork } = props
 
   // Queries
   const { data: networkList = [] } = useGetVisibleNetworksQuery()
 
   // Memos
   const dappSupportedNetwork = React.useMemo(() => {
+    if (!accountId) {
+      return []
+    }
```

Comment:
Does this have a negative effect if there is not yet a `selectedAccountId` for a `CoinType`?